### ISSUE TITLE
Ritual of Magnetism doesn't work on Iron Ore (maybe others).

### ISSUE THREAD

**Issue by spdkils** [talonos2/BlightfallTracker#165]
The blood magic ritual will pull up all kinds of Tin, Copper, etc. However it doesn't work on plain ol' iron.

It pulls up metals that are set much harder to mine, but won't pull iron out of the ground.

**Comment by shade-of-ox** [comment]
That's most likely a bug with UBC Ore Registrar or something, unless your ritual isn't set up properly. I would guess the former, since you're pulling other ores out of the ground. I'm pretty sure it's not a Blood Magic issue, most likely someone just messed up while registering the ores.

On the other hand, why do you need iron specifically? It's dirt common, the tech metals are what you really have trouble finding most of the time. Just go out with a hammer for 10 minutes and you'll have half a stack of ore already.

**Comment by spdkils** [comment]
Because I'm using a decent amount of iron, to complete my biome ritual, to change an area into the nether, so I can move a spawner, and get infinite wither skulls for giggles.

**Comment by spdkils** [comment]
And you're right, none of the iron is registered in the oreDict, while almost all the other ores are. Underground biomes is on Git, and I've been digging thru it.

**Comment by spdkils** [comment]
It takes 6 iron blocks per item I need, so 54 Iron \* 8,  432 Iron....

**Comment by shade-of-ox** [comment]
Gotcha, that makes sense then. So we know what the problem is, now we need
to wait for a fix! :p

I suppose in the meantime you can... I don't know... requisition about a
million iron axes from the quest book and melt them down? ;-;

**Comment by Raclex** [comment]
Use the rail dupe glitch and melt them down :D

**Comment by drageuth** [comment]
Yeah I ran into that one myself.  Considering the last patch was a year ago, I'm not gonna wait around for a patch....  I just went ahead and added the vanilla ores to the ore dictionary myself.  Here's a copy of my fix for ya, just open up your mod folder and put it in the scripts folder:

https://www.dropbox.com/s/kkie81dh7jvifsh/BMagic%20Ore%20Fix.zs?dl=0
